test/base/TestFill.py

```diff
@@ -0,0 +1,277 @@
+from typing import NamedTuple
+import unittest
+import pytest
```

Comment:
You introduce a new requirement here. So far I've told people to just run unittest in /tests, there's now another step. It might be best to add this requirement to requirements.txt then or use plain unittest.

Reply:
Removed dependency on pytest